sdk/eventhub/event-hubs/src/models/public.ts

```diff
@@ -128,6 +128,61 @@ export interface EventHubClientOptions {
   userAgent?: string;
 }
 
+/**
+ * An options bag to configure load balancing settings.
```

Comment:
This configures way more than just load balancing settings.

Reply:
Woops, corrected.